### PR TITLE
add option for calendar weeks to start on Monday instead of Sunday

### PR DESCRIPTION
**Pull request by mikerodonnell89**
#### Please provide a link to the associated issue.

#332

### CODE REVIEW

File: library/src/lib/calendar/calendar.component.ts [hunk 1]
```diff
@@ -100,7 +103,7 @@ export class CalendarComponent implements OnInit, OnChanges, AfterViewChecked {
         'November',
         'December'
     ];
-    weekDays: string[] = ['S', 'M', 'T', 'W', 'T', 'F', 'S'];
+    weekDays: string[];
     daysPerMonth = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
```
Comment (droshev) [inline library/src/lib/calendar/calendar.component.ts]: what if February is 29 days

File: library/src/lib/calendar/calendar.component.ts [hunk 2]
```diff
@@ -694,6 +702,7 @@ export class CalendarComponent implements OnInit, OnChanges, AfterViewChecked {
             this.constructCalendar();
             this.constructCalendarYearsList();
         }
+        this.weekDays = this.mondayStartOfWeek ? ['M', 'T', 'W', 'T', 'F', 'S', 'S'] : ['S', 'M', 'T', 'W', 'T', 'F', 'S'];
```
Comment (droshev) [inline library/src/lib/calendar/calendar.component.ts]: do we need to make this translatable?

File: library/src/lib/calendar/calendar.component.ts (hunk 1 above)
Comment (InnaAtanasova) [reply library/src/lib/calendar/calendar.component.ts]: there is a function `determineDaysInMonth` that determines if Feb has 28 or 29 days

File: library/src/lib/calendar/calendar.component.ts (hunk 2 above)
Comment (mikerodonnell89) [reply library/src/lib/calendar/calendar.component.ts]: I think so, however I think i18n should be a separate story

**Pull request merged**